CHANGELOG.md

```diff
@@ -1,3 +1,128 @@
+# Kubermatic 2.17
+
+## [v2.17.0](https://github.com/kubermatic/kubermatic)
```

Comment:
Please make this a link to the Github release, like for all other release headlines.

Reply:
waiting for v2.17.0 to be released.

Reply:
Done